bundles/http/Citrix/Cherwell/Cherwell.js

```diff
@@ -1211,11 +1181,23 @@ async function buildFilters(functionName, latestSynchronizationTime) {
     return { fieldName, date }
 }
 
-async function listContains(list, obj, property1, property2) {
-    for (const listObj of list) {
-        if (listObj[property1] === obj[property1] && listObj[property2] === obj[property2]) {
-            return true;
+function convertDateProperties(...args) {
+    const propertyNames = [...args]
+    return obj => Object.entries(obj).reduce((acc, [prop, value]) => {
+        if (propertyNames.includes(prop)) {
+            acc[prop] = new Date(value);
+        } else {
+            acc[prop] = value;
+        } return acc;
+    }, {})
+}
+
+function uniqWith(arr, comparator) {
```

Comment:
this is already implemented by Lodash, just replace your `uniqWith` with `_.uniqWith`

Reply:
How do we include the Lodash library on the microapp script?

Reply:
Lodash Implemented